packages/expo/src/Notifications.ts

```diff
@@ -173,6 +184,15 @@ export default {
     _initialNotification = notification;
   },
 
+  // User passes set of actions titles.
+  createCategoryIOSAsync(categoryId: string, actions: Array<ActionType>): Promise<void> {
```

Comment:
Use `[]` syntax for simple arrays:
```suggestion
  createCategoryIOSAsync(categoryId: string, actions: ActionType[]): Promise<void> {
```